server/models/User.js

```diff
@@ -67,5 +67,7 @@ module.exports = function(bookshelf){
       });
   });
 
+  module.exports = bookshelf.model('User', User);
```

Comment:
By all means.
Wait till you see the next arrangement. It will be much more clear. 
But this is almost what you have to do with bookshelf to keep from getting circular model deps.